.github/workflows/ci.yml

```diff
@@ -41,25 +41,43 @@ jobs:
           CARGO_TARGET_DIR="target/clippy" \
           RUSTFLAGS="-D warnings" \
           cargo clippy --all-features --all-targets --locked
+          cd ..
+
+          cd libs/chain-signatures
+          CARGO_TARGET_DIR="target/clippy" \
+          RUSTFLAGS="-D warnings" \
+          cargo clippy --all-features --all-targets --locked
+          cd ../..
```

Comment:
the `../..` at the end of the steps are not necessary.

Reply:
It's just to complete the pattern - I tripped myself by not `cd ..`ing at the end of the previous step :)